packages/sqflite/pubspec.yaml

```
@@ -0,0 +1,20 @@
+name: sqflite_tizen
+description: SQLite plugin for Flutter. This plugin provides the Tizen implementation for SQLite.
+version: 0.1.0
+homepage: https://github.com/flutter-tizen/plugins
```

You may add the repository information like this:

```suggestion
homepage: https://github.com/flutter-tizen/plugins
repository: https://github.com/flutter-tizen/plugins/tree/master/packages/sqflite
version: 0.1.0
```